### ISSUE TITLE
(-215:Assertion failed) VScn::contains(scn) && VDcn::contains(dcn) && VDepth::contains(depth) in function 'cv::CvtHelper

### ISSUE THREAD

**Issue by geekzain** [xuannianz/EfficientDet#75]
Hey ! Could you help me out with this error, I don't know what to do. I followed the steps and when I run the train.py, it gives me this error.
![image](https://user-images.githubusercontent.com/43846739/74448795-ddf53500-4e7b-11ea-99b9-38c8c85be569.png)

**Comment by km23630** [comment]
check your coco data path and parameters.

this is my train parameter and directory structure
Efficient_det\train.py
--snapshot imagenet 
--phi 0 
--gpu 0 
--random-transform 
--compute-val-loss
 --freeze-backbone 
--epochs 50 
--batch-size 8 
--steps 14768 
--workers 8 
coco 
Efficient_det/datasets/coco

==
├Efficient_det
│└datasets
│　└coco
│　　├annotation
│　　└images
│　　　├train2017
│　　　├test2017
│　　　└val2017
└train.py

**Comment by geekzain** [comment]
Thank you, the images folder wasn't separated in train2017/test2017/val2017.
